odbc/src/api/data_tests.rs

```diff
@@ -107,7 +111,7 @@ lazy_static! {
                 "unit_str": "a",
                 "guid": Bson::Binary(Binary {
                     subtype: BinarySubtype::Uuid,
-                    bytes: vec![0u8, 1u8, 2u8, 3u8, 4u8, 5u8, 6u8, 7u8, 9u8, 10u8, 11u8, 12u8, 13u8, 14u8, 15u8],
+                    bytes: vec![0xd5, 0xe3, 0xac, 0x31, 0xfd, 0x29, 0x49, 0xe3, 0x87, 0x59, 0x08, 0xd1, 0x16, 0xe9, 0x8b, 0x29],
```

Comment:
I replaced this with a valid uuid generated from `mongosh`, but was curious if `8u8` was omitted from the original on purpose (which would make it invalid since it needs 16 bytes, and was only 15).